src/redux/sagas/actions/manageVerifiedRecipients.ts

```diff
@@ -45,7 +45,6 @@ function* manageVerifiedRecipients({
     verifiedAddresses = [],
```

Comment:
I think we can actually remove this entire saga and all of its' typedefs

Reply:
Eradicated 👌